test/api/v3/integration/groups/GET-groups_groupId_members.test.js

```diff
@@ -57,6 +66,30 @@ describe('GET /groups/:groupId/members', () => {
     expect(res[0].profile).to.have.all.keys(['name']);
   });
 
+  it('populates only some fields if req.query.includeAllPublicFields === true and it is a guild', async () => {
```

Comment:
If that's the case, you'll need to add the query in your test case:

```js
let res = await user.get(`/groups/${group._id}/members`);
```

Does not include the query parameter.